modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,159 @@
+= Provision a Serverless Cluster
+:description: Learn how to create a Serverless Cloud cluster.
+:page-cloud: true
+:page-beta: true
+
+Redpanda Serverless is the easiest way to start event streaming in the cloud: You can start streaming in 30 seconds. 
+
+== What is Redpanda Serverless?
+
+Redpanda Serverless is a managed streaming service (Kafka API) that completely abstracts users from scaling and operational concerns, and you only pay for what you use. It provides data isolation and data residency:
+
+* Your data is hosted on Redpanda Cloud clusters shared among multiple tenants. Other tenants never access your data.
+* Your data is stored in the Redpanda Cloud account, in the region you select. Data never leaves the selected region.
+
+=== Deployment and networking
+
+Redpanda Serverless currently is available in select regions in AWS. We expect your applications to be deployed in the same AWS region. For best performance, select the region closest to your applications. Serverless is not guaranteed to be pinned to a particular availability zone within that region.
+
+Your applications talk to Serverless over the public internet using encrypted connections. Serverless does not support private networking (that is, VPC peering to talk over private IPs). Use Redpanda Dedicated clusters for private networking.
+
+=== Limitations
+
+Each Serverless cluster has the following limits:
+
+* Ingress: up to 10 MBps, 0.5 MBps guaranteed
+* Egress: up to 30 MBps, 1.5 MBps guaranteed
+* Partitions: 100
+* Max message size: 20 MB
+* Max retention: unlimited
+* Storage: unlimited
+* Users: 30
+* ACLs: 120
+* Consumer groups: 200
+
+=== Serverless vs Dedicated and BYOC
+
+Redpanda Serverless is a good fit for the following:
+
+* Starter and growning workloads
+* Spiky workloads (dev environments, systems that only occasionally get busy, or workloads that come and go)
+* Fast and dynamic cluster creation: you can use a Serverless cluster as an isolated container for topics
+
+Because a Serverless cluster is created in under 1 second, you can surface it in your applications (for example, for tenant isolation).
```

Comment:
```suggestion
Because a Serverless cluster is created in under one second, you can surface it in your applications (for example, for tenant isolation).
```